### PR TITLE
#1897 Remove "workerClusterName" from Pipeline with manager

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1897 )
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/pages/PipelinePage/PipelineNewPage/PipelineNewPage.js [hunk 1]
```diff
@@ -141,7 +141,8 @@ class PipelineNewPage extends React.Component {
   };
 
   fetchWorker = async () => {
-    const { workerClusterName: name } = this.state.pipeline;
+    const { tags } = this.state.pipeline;
+    const { workerClusterName: name } = tags;
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/PipelinePage/PipelineNewPage/PipelineNewPage.js]: workerClusterName is fine here. It's actually more readable though it's a bit verbose

**Comment by konekoya** [comment]
@wu87988622 I think this PR should also add an API test case to make sure the `tags` are in the API response

File: ohara-manager/client/src/components/pages/PipelinePage/PipelineNewPage/PipelineNewPage.js (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/pages/PipelinePage/PipelineNewPage/PipelineNewPage.js]: ｏｋ

**Comment by wu87988622** [comment]
> @wu87988622 I think this PR should also add an API test case to make sure the `tags` are in the API response

ok thanks, i forget....

**Comment by wu87988622** [comment]
retry manager-e2e

**Pull request merged**